queryviews/migrations/0003_combo_1.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.0.4 on 2018-05-17 20:23
```

Comment:
Vinod and I discussed separating them so we can run the first couple on production alone and test them, see how long they take, etc. If they are expensive to build, we might want to do them in steps to avoid a problem rolling them all back at once.